.ci/cico_deploy.sh

```diff
@@ -37,19 +37,28 @@ echo "CHE VALIDATION: Verification skipped until job devtools-che-functional-tes
 # . cico_run_EE_tests.sh
 # echo "CHE VALIDATION: Verification passed. Pushing Che server image to prod registry."
 
+echo "CHE VALIDATION: Verification is disabled. Pushing Che server images to prod registry without verification."
+
 STAGE_IMAGE_TO_PROMOTE="${DOCKER_HUB_NAMESPACE}/che-server-multiuser:${CHE_SERVER_DOCKER_IMAGE_TAG}"
-PROD_IMAGE_DEVSHIFT="push.registry.devshift.net/che/che-multiuser:${CHE_SERVER_DOCKER_IMAGE_TAG}"
 
-echo "CHE VALIDATION: Pushing image ${PROD_IMAGE_DEVSHIFT} to devshift"
+DEVSHIFT_TAG=$(echo $GIT_COMMIT | cut -c1-${DEVSHIFT_TAG_LEN})
+PROD_IMAGE_DEVSHIFT="push.registry.devshift.net/che/che-multiuser:${DEVSHIFT_TAG}"
+PROD_IMAGE_DEVSHIFT_LATEST="push.registry.devshift.net/che/che-multiuser:latest"
 
-if ([ -z "${DEVSHIFT_USERNAME+x}" ] || [ -z "${DEVSHIFT_PASSWORD+x}" ]); then
-    echo "ERROR: Cannot push to registry.devshift.net: credentials are not set. Aborting"
-    exit 1
+if [ -n "${DEVSHIFT_USERNAME}" -a -n "${DEVSHIFT_PASSWORD}" ]; then
```

Comment:
Why do you need to change the logic here?

Reply:
because IMO `if / else` structure is more straight forward and easier to understand